docs/2023/puzzles/day05.md

```diff
@@ -6,6 +6,151 @@ import Solver from "../../../../../website/src/components/Solver.js"
 
 https://adventofcode.com/2023/day/5
 
+## Solution summary
+
+First and foremost, the data must be parsed from the file into the following classes:
+- ```final case class Resource(start: Long, `type`: DestinationEnum)``` the source resource in the first task
+- ```final case class ResourceWithRange(start: Long, end: Long, `type`: DestinationEnum)``` the source resource in the second task, in the first one you can use this same model but set `end = 0`
+- I used the following enumeration for storing the type of resource:
+```scala
+object DestinationEnum extends Enumeration {
```

Comment:
I think it's worth using Scala 3 for blog posts. This particular example would be:
```scala
enum Destination:
  case Seed, Soil, Fertilizer, Water, Light, Temperature, Humidity, Location
```